release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -0,0 +1,896 @@
+
+
+[id="ocp-4-12-add-on-support-status"]
+== {product-title} layered and dependent component support and compatibility
+
+The scope of support for layered and dependent components of {product-title} changes independently of the {product-title} version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+
+[id="ocp-4-12-new-features-and-enhancements"]
+== New features and enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[id="ocp-4-12-rhcos"]
+=== {op-system-first}
+
+[id="ocp-4-12-installation-and-upgrade"]
+=== Installation and upgrade
+
+[id="ocp-4-12-post-installation"]
+=== Post-installation configuration
+
+[id="ocp-4-12-web-console"]
+=== Web console
+
+[id="ocp-4-12-oc"]
+=== OpenShift CLI (oc)
+
+[id="ocp-4-12-ibm-z"]
+=== IBM Z and LinuxONE
+
+[id="ocp-4-12-ibm-power"]
+=== IBM Power
+
+[id="ocp-4-12-security"]
+=== Security and compliance
+
+[id="ocp-4-12-networking"]
+=== Networking
+
+[id="ocp-4-12-storage"]
+=== Storage
+
+[id="ocp-4-12-registry"]
+=== Registry
+
+[id="ocp-4-12-olm"]
+=== Operator lifecycle
+
+[id="ocp-4-12-osdk"]
+=== Operator development
+
+[id="ocp-4-12-jenkins"]
+=== Jenkins
+
+[id="ocp-4-12-machine-api"]
+=== Machine API
+
+[id="ocp-4-12-machine-config-operator"]
+=== Machine Config Operator
+
+[id="ocp-4-12-nodes"]
+=== Nodes
+
+[id="ocp-4-11-logging"]
+=== Logging
+
+[id="ocp-4-12-monitoring"]
+=== Monitoring
+
+[id="ocp-4-12-scalability-and-performance"]
+=== Scalability and performance
+
+[id="ocp-4-12-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-12-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-12-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+
+// Note: use [discrete] for these sub-headings.
+
+[id="ocp-4-12-deprecated-removed-features"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the table, features are marked with the following statuses:
+
+* *GA*: _General Availability_
+* *DEP*: _Deprecated_
+* *REM*: _Removed_
+
+//TODO: remove anything that has been REM since 4.7?
+
+.Deprecated and removed features tracker
```

Comment:
Were you going to update the framework of this table to support 4.12?

Reply:
I was thinking of updating this at a later date. I am not really 100% on what should stay and what should go. Like should I recycle the dep/removal announcements or remove them since they were in 4.11 RNs? What are your thoughts @kalexand-rh

Reply:
Ack! Thank you!